_posts/2017-04-13-sketching-in-code.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Sketching in code to build online services"
+category: [blog]
+tags: 'technology'
+author: Simon Schwartz
+author-excerpt: "Simon Schwartz is a frontend developer at the Digital Transformation Agency."
+thumbnail: /images/blog-thumbnails/Sketch_in_code_sml.png
+hero-image: /images/blog-content/Sketch_in_code_hero.png
+---
+
+!Four developers working together in an office.]({{ site.url }}{{ site.baseurl }}{{ page.hero-image }})
+
+*Caption: DTA developers at work.*
+
+A great online service is the result of bringing together excellent design and code. To deliver these services it is critical designers and developers work together. 
+
+It is no use if a designer gives a developer a set of high fidelity specs that don’t consider the constraints of the technology the product is being built in, or the browsers it will be displayed on.
+
+Luckily, this is a pretty easy problem to solve—allow designers and developer to work closely together, and sketch in code. At the DTA, this approach is BAU, and produces great results.
+
+## Ideas are cheap, actually making stuff is where you begin to win the battle
+
+It is good practice to test your ideas early. By doing this you are able to identify some of the constraints you may face—whether it is navigation, layout, accessibility or mobile responsiveness. The best way to do this is by sketching in code, prototyping your work early and iterating quickly.
```

Comment:
Content guide says to use [spaced em-dashes](http://content-guide.apps.staging.digital.gov.au/punctuation-grammar/#em-dashes-mdash)